sources/packages/web/src/components/common/OfferingSummary.vue

```diff
@@ -130,6 +184,17 @@ export default defineComponent({
     const loading = ref(false);
     const searchBox = ref("");
     const { dateOnlyLongPeriodString } = useFormatters();
+    const intensityFilter = ref("");
+    const startDate = ref("");
+    const endDate = ref("");
+    const startDateMenu = ref(false);
+    const endDateMenu = ref(false);
+    const searchOfferingsForm = ref({} as VForm);
+
+    function capitalizeFirstWord(str: string) {
```

Comment:
Can we use a style class like text-uppercase here or do we really need to to the first word?